static/js/flow/emailTypes.js

```diff
@@ -11,12 +11,13 @@ export type EmailInputs = {
 export type EmailValidationErrors = EmailInputs;
 
 export type EmailState = {
-  inputs:           EmailInputs,
-  subheading:       ?string,
-  params:           Object,
-  validationErrors: EmailValidationErrors,
-  sendError:        EmailSendError,
-  fetchStatus?:     ?string,
+  inputs:                  EmailInputs,
+  subheading:              ?string,
+  params:                  Object,
+  validationErrors:        EmailValidationErrors,
+  sendError:               EmailSendError,
+  fetchStatus?:            ?string,
+  supportsAutomaticEmails?: boolean,
```

Comment:
can you fix the alignment of these props? you can use the `column` utility to automatically align them (`column -t` does a table layout).